android/app/src/main/res/raw/keep.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools"
+    tools:keep="@mipmap/ic_launcher, @drawable/bootsplash*, @drawable/alert_background,
+        @drawable/ic_launcher*, @drawable/ic_notification*, @drawable/picker_background,
+        @drawable/rn_edit_text_material" />
```

Comment:
This is a small selection of resources, did we get this list from anywhere specific?

Reply:
I inspected the res directory and all the resources that were used in the codebase by search, were added in keep rule. For example, `drawable/datepicker_background.xml` wasn't being referenced anywhere, so I didn't add it to the keep rule. 

So there's no prescription that says what to keep but anything that's referenced and is being used can be kept. I also tested on build variants like production and adhoc and I get the correct assets as our keep xml file is inherited for each build variant.

https://developer.android.com/build/shrink-code#keep-resources

<img width="429" alt="Screenshot 2024-01-11 at 6 08 06 PM" src="https://github.com/Expensify/App/assets/47336142/e23fce4f-f606-40fc-b144-195bb959f62f">

Reply:
Resources in `react-native/assets` aren't touched here, so that's why they aren't added in the keep rules.

Reply:
> I also tested on build variants like production and adhoc and I get the correct assets as our keep xml file is inherited for each build variant.

Okay cool, that's what I was worried about as it's not clear that alternative resourses would be kept. For example: `drawable-xxxhdpi/bootsplash_logo.png`

I'm just generating another test build to confirm that we're not defaulting to the lower res drawables.